glossary.md

```diff
@@ -0,0 +1,89 @@
+# Glossary of Optimistic Ethereum Terms
+
+### Layer 1 (L1)
+
+The Ethereum Mainnet.
+
+### Layer 2 (L2)
+
+The Optimistic Ethereum Network.
```

Comment:
A network that derives its security from a layer one

Reply:
```suggestion
A network that derives its security from a layer one.
```